src/decide.js

```diff
@@ -45,7 +44,7 @@ export class Decide {
         this.instance.sessionRecording.afterDecideResponse(response)
         autocapture.afterDecideResponse(response, this.instance)
 
-        parseFeatureFlagDecideResponse(response, this.instance.persistence)
+        this.instance.featureFlags.receivedFeatureFlags(response)
```

Comment:
thinking out loud, not a leading question: this feels like a change in behaviour that's worth checking... Do we guarantee we only make one decide call? Since we now call `receivedFeatureFlags` for any decide call not just the first. Do we we risk starting calling someone's event handler multiple times and changing their application behaviour?

Should we ensure it is only the first decide response?

Non-blocking since I'm thinking out loud :)

Reply:
I do like the change to reduce coupling between decide and featureflag though!

Reply:
Viewing our site we only call decide once... Maybe I was thinking of our Cypress tests 😵‍💫

Reply:
And docs in code on `onFeatureFlags` say "will be called once the feature flags are ready or when they are updated."

So, I've answered my own out-loud-thinking, it's expected behaviour to call feature flag handlers multiple times